### ISSUE TITLE
Why change $http.defaults.headers.post?

### ISSUE THREAD

**Issue by amit777** [nraboy/ng-cordova-oauth#228]
I noticed a pattern in the lib where this line of code is called:

```
 $http.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
``

Why not just set the Content-Type header within the $http() request?  I have not run into an issue yet, but it seems like there is potential conflict if you have existing code that needs to post with a default content-type.

**Comment by matheusrocha89** [comment]
Already merged #220 I think it's already in the last version.

**Comment by nraboy** [comment]
Merged into the dev branch. Will be in staging today. Master will be a while to confirm staging works
